compiler/llvm/clangUtil.cpp

```diff
@@ -4712,8 +4712,14 @@ static void makeBinaryLLVMForHIP(const std::string& artifactFilename,
   // So this loop should run exactly one iteration.
   INT_ASSERT(gpuArches.size() == 1);
 
-  std::string targets;
-  std::string inputs;
+  std::string targets = "-targets=host-x86_64-unknown-linux";
```

Comment:
I'm surprised to see a fixed linux64 target here. Have we just been lucky to only build on x86_64 machines?